mw-webapp/src/component/checkbox/Сheckbox.tsx

```diff
@@ -0,0 +1,83 @@
+import {Indicator, Root} from "@radix-ui/react-checkbox";
+import {CheckIcon} from "@radix-ui/react-icons";
+import clsx from "clsx";
+import styles from "src/component/checkbox/Сheckbox.module.scss";
+
+/**
+ * Checkbox props
+ */
+interface CheckboxProps {
+
+  /**
+   * Custom class for the checkbox.
+   */
+  className?: string;
+
+  /**
+   * Unique identifier for the checkbox.
+   */
+  id: string;
+
+  /**
+   * Content for the label.
+   */
+  label: string;
+
+  /**
+   * The name of the checkbox.
+   */
+  name?: string;
+
+  /**
+   * The value of the checkbox.
+   * @default "on"
+   */
+  value?: string;
+
```

Comment:
Can you pls explain what doing this prop? Why default value is on? How it depends on checkbox?

Reply:
this prop is not needed in the project at this stage, I removed it